timeseries/src/autogluon/timeseries/models/gluonts/models.py

```diff
@@ -190,6 +191,51 @@ class SimpleFeedForwardModel(AbstractGluonTSModel):
     gluonts_estimator_class: Type[GluonTSEstimator] = SimpleFeedForwardEstimator
 
 
+class TemporalFusionTransformerModel(AbstractGluonTSModel):
+    """TemporalFusionTransformer model for forecasting, as described in [Lim2020]_.
+
+    .. [Lim2021] Lim, Bryan, et al.
+        "Temporal Fusion Transformers for Interpretable Multi-horizon Time Series Forecasting."
+        International Journal of Forecasting. 2021.
+
+    See `AbstractGluonTSModel` for common parameters.
+
+    Other Parameters
+    ----------------
+    context_length : int or None, default = None
+        Number of past values used for prediction.
+        (default: None, in which case context_length = prediction_length)
+    hidden_dim : int, default = 32
+        Size of the hidden layer.
+    num_heads : int, default = 4
+        Number of attention heads in multi-head attention.
+    dropout_rate : float, default = 0.1
+        Dropout regularization parameter
+    batch_size : int, default = 32
+        Size of the mini-batch during training.
+    epochs : int, default = 100
+        Number of epochs the model will be trained for.
+    """
+
+    gluonts_estimator_class: Type[GluonTSEstimator] = TemporalFusionTransformerEstimator
+
+    def _get_estimator(self) -> GluonTSEstimator:
+        """Return the GluonTS Estimator object for the model"""
+        hyperparameters = self._get_estimator_init_args()
+        # Turning off hybridization prevents MXNet errors when training on GPU
+        hyperparameters["hybridize"] = False
+        # TFT cannot handle arbitrary quantiles, this is a workaround
+        hyperparameters["num_outputs"] = 9
+        supported_quantiles = set([0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9])
+        if not set(self.quantile_levels).issubset(supported_quantiles):
```

Comment:
cool. so if `self.quantile_levels=[0.1, 0.5, 0.9]`, the current GluonTS logic makes sure these are returned? Are the default tests covering this?